### ISSUE TITLE
[MU4 Task] [Accessibility] Use instrument description as accessible description in New Score dialog

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#13600]
**Task description**
We display instrument descriptions in the bottom left corner of the New Score dialog. These descriptions should be spoken by screen readers after the name of the selected instrument.

![image](https://user-images.githubusercontent.com/11011881/193605480-fd0df5bf-10e5-4d3a-8fba-824dd701cbe7.png)

This involves setting the accessible description of items in the Instruments column to the instrument description text. For example:

- **Accessible name:** _"Quijada"_
- **Accessible description:** _"Dried donkey or horse jawbone, traditionally used as a rattle in some Latin American countries."_

**Comment by miiizen** [comment]
I can take a look at this!
